src/Page/Donate/styles.css

```
@@ -0,0 +1,292 @@
+
+.donate-page {
+  background-color: #fff;
+  text-align: center;
+}
+.donate-page .page-content {
+  width: 100%;
+}
+
+.partition-line {
+  border-top: 2px solid #cdcdcd;
+}
+.donate-page-section {
+  margin-bottom: 64px;
+}
+.donate-page-image-section {
+  margin: 40px 0;
+  background-image: url("./assets/bg.png");
+  height: 800px;
+  width: 100%;
+  background-position: center;
+  background-repeat: none;
```

`none` is not a valid value for `background-repeat`.
```suggestion
  background-repeat: no-repeat;
```